en/docs/administer/wso2-administration-guide.md

```diff
@@ -1,5 +1,11 @@
 # Administration Guide
 
 
-This section provides instructions on how to configure WSO2 Identity Server.
+The administration guide provides you in-depth instructions on how to configure WSO2 Identity Server.
+
+If you haven’t installed the product yet, see [installing the product](../../setup/installing-the-product) so that you can proceed with the configurations. 
```

Comment:
haven't --> have not

see [installing the product](../../setup/installing-the-product) to install the product and proceed with the configurations.